sktime/forecasting/model_evaluation/_functions.py

```diff
@@ -28,6 +30,7 @@ def evaluate(
     X=None,
     strategy="refit",
     scoring=None,
+    error_score=np.nan,
```

Comment:
is this what it is called in sklearn? If not, would `score_on_exception` or similar be better?

Reply:
`error_score=np.nan,` is from `sklearn`